game.rb

```diff
@@ -1,80 +1,103 @@
-@players = []
-@current_position = []
-@snake_pos =[[37,1],[25,5],[47,9],[65,59],[99,62],[87,54],[92,69]]
-@ladder_pos = [[2,60],[6,27],[11,51],[35,56],[63,96],[68,88]]
-puts "Enter number of players"
-number=gets.chomp.to_i
-i=1
-var ='a'
-while i <= number
-  @players << var
-  @current_position << 0
-  i=i+1
-  var = var.next
-end
-i=0
-def move(p)
-  position = p
-  temp = 1+ rand(6)
-  puts "The dice rolled #{temp}"
-  position = position + temp
-  if position > 100
-    print("Sorry. This move cannot be counted")
-    position-=temp
-  end
+class Play
 
-  if position == 100
-    return position
+  def initialize(players)
+    @arr = []
+    @arr = players
+    @snake_pos =[[37,1],[25,5],[47,9],[65,59],[99,62],[87,54],[92,69]]
```

Comment:
you can make this getter setter too? hash would be more preferable here over the two dimensional array.